setup.py

```diff
@@ -56,5 +53,10 @@
         'Topic :: Terminals :: Serial',
     ],
     platforms = 'any',
-    scripts = ['serial/tools/miniterm.py'],
+    entry_points={
+        'console_scripts': [
+            'list_serial_ports = serial.tools.list_ports:main',
```

Comment:
why add `list_ports`? it can be called using `python -m serial.tools.list_ports`? I even considered removing miniterm from the scripts as it can be called via `-m` just as easily (left it in because of backward compatibility).